README.md

```diff
@@ -1,5 +1,58 @@
-# ssi-dim-wallet-stub
+# SSI DIM Wallet stub application
 
-Stub application for Decentralize Identity Management (DIM) Wallet
+#### Note: *This application can not be used in production. This is designed and developed for local and integration testing only*
 
-### WIP
+The SSI Decentralize Identity Management (DIM)
```

Comment:
I think it should be called "Decentralized", the line break should be removed and might be good to explain the abbreviation SSI as well.

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/006e091ce7de7a4c265534389e3288e35968c732